pandas/core/categorical.py

```diff
@@ -2013,7 +2014,20 @@ def repeat(self, repeats, *args, **kwargs):
 # The Series.cat accessor
 
 
-class CategoricalAccessor(PandasDelegate, NoNewAttributesMixin):
+@accessors.wrap_delegate_names(delegate=Categorical,
```

Comment:
an alternative way to do this is to add in the classes themselves which methods / properties should be considered accessors (either in a class method or via decorators), i did do this for a while, but actually though it simpler here. but can revisit at a later point.